packages/manager/modules/overthebox/src/overthebox/actions/overTheBox-actions.html

```diff
@@ -0,0 +1,80 @@
+<div class="telecom-overTheBox-actions">
+    <div class="telecom-section-content">
+        <oui-message type="error" dismissable data-ng-if="$ctrl.isError">
+            <span data-ng-bind="$ctrl.errorMessage"></span>
+        </oui-message>
+
+        <div class="widget-presentation">
+            <h2
+                class="widget-presentation-title"
+                data-translate="overTheBox_actions_list"
+            ></h2>
+
+            <div
+                class="oui-datagrid-topbar__actions"
+                data-ng-if="!$ctrl.isLoading"
+            >
+                <button
+                    type="button"
+                    class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+                    data-ng-click="$ctrl.getActions()"
+                    title="{{'common_refresh' | translate}}"
```

Comment:
Could you please avoid using translation from a common location?

Since you are creating a new isolated module, then it can also bring its own
translations.

That would be really well appreciate.